### ISSUE TITLE
TabItem Control don't show Header content

### ISSUE THREAD

**Issue by DrCQ** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2638]
I use current version 4.4.0 with WPF for NET 6.0.
The TabItem control don't show the Header content, probably due to wrong or missing Foreground property value. Adding the Header inside the TabItem also don't solve the problem. First with explicit assignment of Foreground value the Header content is visible.
` <TabItem>
    <TabItem.Header>
       <TextBlock Text="Header Capture" Foreground="{DynamicResource MaterialDesignBody}"/>
    </TabItem.Header>
     
</TabItem>`

**Comment by rasoulia** [comment]
same as me. use materialdesignextensions and then set 
`Style="{StaticResource MaterialDesignTabControl}"`

also add this line to App.xaml
```
<ResourceDictionary Source="pack://application:,,,/MaterialDesignExtensions;component/Themes/Generic.xaml" />
<ResourceDictionary Source="pack://application:,,,/MaterialDesignExtensions;component/Themes/MaterialDesignDarkTheme.xaml" />
```

but we have to wait for fix it without using MaterialDesignExtensions

**Comment by Keboo** [comment]
This is fixed and will be out in the next 4.5.0 preview nightly release.
